test/functionalTest/cases/3045_wrong_inclusion_location_metadata_notification/wrong_inclusion_location_metadata_V1.test

```diff
@@ -0,0 +1,458 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This test is covering this case

![imagen](https://user-images.githubusercontent.com/1534240/48831269-c4e3fe00-ed76-11e8-8797-c09f578604e5.png)

Great!

NTC

Reply:
It's ok.